packages/salesforcedx-vscode-core/src/commands/forceOrgCreate.ts

```diff
@@ -13,15 +13,22 @@ import * as path from 'path';
 import * as vscode from 'vscode';
 import { nls } from '../messages';
 import {
+  CancelResponse,
+  CompositeParametersGatherer,
+  ContinueResponse,
   FileSelection,
   FileSelector,
+  ParametersGatherer,
   SfdxCommandlet,
   SfdxCommandletExecutor,
   SfdxWorkspaceChecker
 } from './commands';
 
-class ForceOrgCreateExecutor extends SfdxCommandletExecutor<FileSelection> {
-  public build(data: FileSelection): Command {
+export const DEFAULT_ALIAS = 'scratchOrg';
+export class ForceOrgCreateExecutor extends SfdxCommandletExecutor<
+  FileSelection & Alias
```

Comment:
Why is it the intersection rather than the union? Also, you should define a type alias instead of repeating `FileSelection & Alias` everywhere.

Reply:
Union will OR the interfaces together which is not correct. The ContinueResponse should always have the fields from both interfaces since both a config file and alias are required.

I will make the type alias